Looms.PoS/Looms.PoS.Application/Features/Reservation/Commands/UpdateReservation/UpdateReservationRequestValidator.cs

```diff
@@ -28,21 +28,9 @@ public UpdateReservationRequestValidator(IServicesRepository servicesRepository)
         RuleFor(x => x.ServiceId)
             .MustBeValidGuid()
             .MustAsync(async (serviceId, cancellation) => 
-                {
-                    if (Guid.TryParse(serviceId, out var guid))
-                    {
-                        try
-                        {
-                            await _servicesRepository.GetAsync(guid);
-                            return true;
-                        }
-                        catch (LoomsNotFoundException)
-                        {
-                            return false;
-                        }
-                    }
-                    return false;
-                })
+            {
+                return Guid.TryParse(serviceId, out var guid) && await _servicesRepository.GetAsync(guid) != null;
```

Comment:
CascadeMode.Stop and CustomAsync just call the GetAsync method

Reply:
i do not really get what you want but i changed it?

Reply:
there is no need to do `Guid.TryParse`, that is what `.MustBeValidGuid()` is for

Reply:
changed it